### PR TITLE
put back 4.0 tag images

### PR DESCRIPTION
**Pull request by enxebre**
Since we merge https://github.com/openshift/cluster-api-provider-aws/pull/81 and https://github.com/openshift/cluster-api-provider-libvirt/pull/73
They will become part of the next payload. This puts back the 4.0 tag images as we want the payload build process to replace the images name value with the latest hash generated from master for both actuators

### CODE REVIEW

**Comment by enxebre** [comment]
cc @paulfantom @ingvagabund

File: install/0000_50_machine-api-operator_01_images.configmap.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ metadata:
...
+  images.json: '{"clusterAPIControllerAWS": "docker.io/openshift/origin-aws-machine-controllers:v4.0.0", "clusterAPIControllerManagerAWS": "docker.io/openshift/origin-aws-machine-controllers:v4.0.0", "clusterAPIControllerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0", "clusterAPIControllerManagerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0", "clusterAPIControllerManagerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0", "clusterAPIControllerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0"}'
```
Comment (ingvagabund) [inline install/0000_50_machine-api-operator_01_images.configmap.yaml]: What a shame the JSON is not formatted :P

**Comment by ingvagabund** [comment]
Looks good, don't see anything wrong about it. As long as the e2e is green (verifying the images work with the CRD), we can merge.

**Comment by ingvagabund** [comment]
/approve

Keeping lgtm for @paulfantom

**Pull request merged**